tests/models/dsheetpiling/test_dsheetpiling_model.py

```diff
@@ -229,21 +242,36 @@ def test_execute_console_successfully(self):
         test_folder = Path(TestUtils.get_local_test_data_dir("dsheetpiling"))
         test_file = test_folder / "bm1-1.shi"
         output_test_folder = Path(TestUtils.get_output_test_data_dir("dsheetpiling"))
-        output_test_file = output_test_folder / "test.shi"
-
-        df.parse(test_file)
-        df.serialize(output_test_file)
+        serialized_input_test_file = output_test_folder / "test.shi"
 
         # 2. Verify initial expectations.
-        assert output_test_file.is_file()
+        assert test_file.is_file()
 
         # 3. Run test.
-        df.filename = output_test_file
+        df.parse(test_file)
+        df.serialize(serialized_input_test_file)
+        assert serialized_input_test_file.is_file()
+        df.filename = serialized_input_test_file
         df.execute()
 
         # 3. Verify return code of 0 (indicates succesfull run)
         assert df.datastructure
 
+    @pytest.mark.acceptance
+    @only_teamcity
```

Comment:
Why does this test runs only under teamcity?

Reply:
I believe this is to avoid them running in GitHub. I will have a look either way.

Reply:
Confirmed. Because they are long time runs is better to avoid running them in GitHub.